addons/sourcemod/scripting/surftimer/replay.sp

```diff
@@ -74,6 +74,10 @@ public void StartRecording(int client)
 {
 	if (!IsValidClient(client) || IsFakeClient(client))
 		return;
+	
+	StopRecording(client);
```

Comment:
are you sure this is a good idea?

I think only the return is within the if condition above and as it is now it will stop recording as soon as it starts recording everytime.